server/channels/app/channel.go

```diff
@@ -2408,7 +2408,6 @@ func (a *App) PostAddToChannelMessage(c request.CTX, user *model.User, addedUser
 		Message:   message,
 		Type:      postType,
 		UserId:    user.Id,
-		RootId:    postRootId,
```

Comment:
Why remove the `RootId`?

Reply:
@kostaspt This change is affecting only the system join and leave messages. They don't need a root ID because we don't want them to appear as a comment on a thread.

Reply:
Ah yes I see 👍🏼